packages/tsconfig-reference/copy/id/categories/Basic_Options_6172.md

```diff
@@ -0,0 +1,5 @@
+---
+display: "Opsi Proyek"
+---
+
+Pengaturan-pengaturan berikut digunakan untuk mendefinisikan tujuan waktu proses (_runtime_) dari proyek Anda, bagaimana dan dimana Anda ingin JavaScript dihasilkan dan level integrasi yang Anda inginkan dengan kode JavaScript yang telah ada.
```

Comment:
Dilihat dari bahasa asalnya 'These settings are used to define the **runtime expectations of your project**, how and where you want the JavaScript to be **emitted**...' 

_runtime_ disini menurutku [target kompilasi](https://github.com/tsconfig/bases/), seperti _node.js_ berdasarkan versi dan  _browser_

Reply:
Ah iya, berarti nanti kurang lebih terjemahannya seperti ini ya

```markdown
Pengaturan-pengaturan berikut digunakan untuk mendefinisikan target kompilasi dari proyek Anda, ...
```